zerver/models/realms.py

```diff
@@ -768,6 +771,21 @@ class Realm(models.Model):  # type: ignore[django-manager-missing] # django-stub
                 SystemGroups.NOBODY,
             ],
         ),
+        can_create_groups=GroupPermissionSetting(
+            require_system_group=True,
+            allow_internet_group=False,
+            allow_owners_group=True,
+            allow_nobody_group=False,
+            allow_everyone_group=False,
+            default_group_name=SystemGroups.MEMBERS,
+            id_field_name="can_create_groups_id",
+            allowed_system_groups=[
+                SystemGroups.MODERATORS,
+                SystemGroups.ADMINISTRATORS,
+                SystemGroups.OWNERS,
+                SystemGroups.MEMBERS,
```

Comment:
Do we not want to provide full-members group as option? If yes, then we can keep this list empty, as this was added to for cases where we want to allow only a couple of system groups and we can avoid adding boolean variables for each system group.

Reply:
fixed!